server/container/guice/cassandra-guice/src/test/java/org/apache/james/CassandraJamesServerTest.java

```diff
@@ -61,11 +62,16 @@ protected void configure() {
                     @Provides
                     @Singleton
                     Session provideSession(CassandraModule cassandraModule) {
-                        CassandraCluster cassandra = CassandraCluster.create(cassandraModule);
```

Comment:
Because in one of our test, when playing the different tests, the DB was not cleaned leading to failing tests.